core/arch/instr_shared.c

```diff
@@ -1266,6 +1266,19 @@ instr_length(dcontext_t *dcontext, instr_t *instr)
     return private_instr_encode(dcontext, instr, false /*don't need to cache*/);
 }
 
+instr_t *
+instr_set_encoding_hint(instr_t *instr, dr_encoding_hint_type_t hint)
```

Comment:
So the arm builds did not fail due to missing symbols before b/c there is no call to this function on arm?

Reply:
Did not fail, also baffled me. I did not debug why, prob. the compiler can remove the function completely, yes.